website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: KMIP - Profiles Support
+description: |-
+  The KMIP profiles define the use of KMIP objects, attributes, operations, message elements
+  and authentication methods within specific contexts of KMIP server and client interaction.
+  These profiles define a set of normative constraints for employing KMIP within a particular
+  environment or context of use.
+---
+
```

Comment:
What about add an introductory paragraph that says that Vault implements version 1.4 of the spec?